app/models/order.rb

```diff
@@ -1,3 +1,6 @@
 class Order < ApplicationRecord
   has_many :order_items
+
+  validates :address, :name, :cc_num, :cvv_code, :zip, presence: true
```

Comment:
Looks good for now, later we can think about CC validations. For cvv_code and zip maybe we could add a validation to accept only integers like ```numericality: {only_integer: true}```